core/src/components/radio/radio.tsx

```diff
@@ -141,6 +143,12 @@ export class Radio implements ComponentInterface {
 
   componentWillLoad() {
     this.emitStyle();
```

Comment:
The contents of `emitStyle` should only fire with the legacy control

Reply:
Should this apply to `ion-toggle` as well or does `ion-toggle` behave directly for style emissions than `ion-checkbox`/`ion-radio`?

Reply:
Ah good catch - toggle should have that logic too. I'll make a PR for that

Reply:
Addressed the extra margin + `emitStyle`.

Still investigating the left margin not selecting the radio. This appears the legacy format as well + looks to apply to the modern format of `ion-toggle` as well.